docs/development.rst

```diff
@@ -94,10 +95,13 @@ Run the Tests
 #. Install test requirements::
 
     pip install -r requirements-test.txt
+    npm install -g gulp
```

Comment:
> Installing it globally allows it to be used as a command.

Yeah, if you just install it locally, you can still call it as a command, as long as you put it in a package.json `script` since npm will automatically add "./node_modules/.bin/" to the `$PATH`.

---

So my current proposal is to drop Gulp and just use ESLint directly, like so:

```json
{
  "dependencies": {
    "yuglify": "^0.1.4"
  },
  "devDependencies": {
    "eslint": "^2.3.0"
  },
  "scripts": {
    "test": "eslint ."
  }
}
```

Then change these docs to just say `npm install` on L99 below and `npm test` on L104.
Plus, remove the **gulp**, **gulp-jshint**, and **jshint** dependencies.